reference/append.rst

```diff
@@ -3,29 +3,60 @@
 Append Distribution
 ###################
 
-Append distributed tables are best suited to append-only event data
-which arrives in a time-ordered series. In the next few sections, we describe how
-users can create append distributed tables, load data into them and also expire
-old data from them.
+While Citus' most common use cases involve hash data distribution,
```

Comment:
Can we add a ``NOTE`` here, with some text around append being a very specialized method (or some such) for further discouragement?